app/src/main/res/layout/widget_entry.xml

```diff
@@ -4,29 +4,34 @@
     android:layout_height="wrap_content"
     android:layout_width="match_parent"
     android:orientation="horizontal"
-    android:padding="4dp">
+    android:padding="@dimen/widget_note_list_entry_layout_padding">
```

Comment:
What is the advantage of moving these dimensions to `dimens.xml`? As far as I can see, you are not re-using any of them! From my point of view, this makes everything much more complex ...